### PR TITLE
Unit tests for OSclients

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag @chemaper 

### DESCRIPTION
- Adding more unittests to cover osclients script.
- removing unnecessary omit

### CODE REVIEW

**Comment by flopezag** [comment]
Need to merge #14  due to a lot of changes are doing in the code which involve a merge of this branch...

**Comment by jframos** [comment]
Conflicts with develop branch.

File: tests/units/test_osclients.py [hunk 1]
```diff
@@ -121,3 +167,236 @@ def test_get_novaclient(self):
...
+
+    def test_set_credential_to_osclients(self):
+        """test_set_credential_to_osclients check that we could set credentials using method set_credential"""
+        username = "new_user"
+        password = "new_password"
+        tenant_name = "new_user cloud"
+        tenant_id = "00000000000000000000000000000002"
+        trust_id = "randomid0000000000000000000000001"
+
+        # FIST CHECK: Credentials from ENV
```
Comment (flopezag) [inline tests/units/test_osclients.py]: FIST or FIRST?

File: tests/units/test_osclients.py [hunk 2]
```diff
@@ -121,3 +167,236 @@ def test_get_novaclient(self):
...
+        osclients.use_v3 = False
+        novaclient = osclients.get_novaclient()
+        self.assertIsNotNone(osclients._session_v2)
+        osclients.set_credential(username, password)
+        self.assertIsNone(osclients._session_v2)
+
+    def test_set_region(self):
+        """test_set_region check that we could change the region after create the client"""
+
+        # FIST CHECK: Region is recovered from ENV
```
Comment (flopezag) [inline tests/units/test_osclients.py]: FIST or FIRST?

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**